tests/Integration/Behaviour/Features/Context/Domain/CartFeatureContext.php

```diff
@@ -196,15 +197,21 @@ public function updateProductQuantityInCart(int $quantity, string $productName,
     }
 
     /**
-     * @When I add :quantity customized products with reference :productReference to the cart :reference
+     * @When I add :quantity customized products with its customization with reference :productReference to the cart :reference
      */
-    public function addCustomizedProductToCarts(int $quantity, $productReference, $reference)
-    {
+    public function addCustomizedProductToCartsWithCustomization(
+        int $quantity,
+        string $productReference,
+        string $reference
+    ) {
         $productId = (int) Product::getIdByReference($productReference);
         $product = new Product($productId);
-        $customizationFields = $product->getCustomizationFieldIds();
+        $customizationsFields = $product->getCustomizationFields();
```

Comment:
Why did you change this? The `$customizationsFields` is only used once to check if it's empty and then you use `$product->getCustomizationFieldIds()` like before?

Reply:
Fixed